docs/querying/sql-functions.md

```diff
@@ -280,27 +295,72 @@ Joins all elements of `arr` by the delimiter specified by `str`.
 
 ## ASIN
 
-`ASIN(<NUMERIC>)`
+Calculates the arc sine of a numeric expression.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `ASIN(expr)`
+* **Function type:** Scalar, numeric
 
-Calculates the arc sine of a numeric expression.
+<details><summary>Example</summary>
 
-## ATAN
+The following example calculates the arc sine of one.
 
-`ATAN(<NUMERIC>)`
+```sql
+SELECT ASIN(1) AS "arc_sine"
+```
+Returns the following:
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+| `arc_sine` |  
+| -- | 
+| `1.5707963267948966` |
+</details>
+
+[Learn more](sql-scalar.md#numeric-functions)
+
+## ATAN
 
 Calculates the arc tangent of a numeric expression.
```

Comment:
Why are some functions explicitly called out as `trigonometric` (line 577) and others not?

Reply:
Agreed to keep as is to match definitions on [scalar functions page](https://druid.apache.org/docs/latest/querying/sql-scalar/#numeric-functions)